docs/releasehistory.rst

```diff
@@ -11,6 +11,34 @@ Releases follow the ``major.minor.micro`` scheme recommended by `PEP440 <https:/
 Current Development
 -------------------
 
+New features
+""""""""""""
+- `PR #433 <https://github.com/openforcefield/openforcefield/pull/433>`_: Closes
+  `Issue #25 <https://github.com/openforcefield/openforcefield/issues/25>`_ by adding
+  initial support for the
+  `LibraryCharges tag in the SMIRNOFF specification <https://open-forcefield-toolkit.readthedocs.io/en/latest/smirnoff.html#librarycharges-library-charges-for-polymeric-residues-and-special-solvent-models>`_
+  using
+  :py:class:`LibraryChargeHandler <openforcefield.typing.engines.smirnoff.parameters.LibraryChargeHandler>`.
+  For a molecule to have charges assigned using LibraryCharges, all of its atoms must be covered by
+  at least one LibraryCharge. If an atom is covered by multiple LibraryCharges, then the last
+  one read will be applied. At this time, there is no concept of "residues" during parametrization,
```

Comment:
It's unclear if this means we can parameterize a biomolecule made up of a few "residues" or not. Can you clarify whether or not it is possible to provide a library of residue charge definitions and have the whole protein charged this way, provided all atoms have one LibraryCharge applied?

Reply:
Good question. I've added text to address this directly: 

>   This functionality is thus able to apply per-residue charges similar to those in traditional
>   protein force fields. At this time, there is no concept of "residues" or "fragments" during
>   parametrization, so it is not possible to assign charges to `some` atoms in a molecule using
>   LibraryCharges, but calculate charges for other atoms in the same molecule using a different
>   method. To assign charges to a protein, LibraryCharges SMARTS must be provided for
>   the standard residues, as well as for any capping groups and post-translational modifications 
>   that are present.